src/ci/bin/test.inc.sh

```diff
@@ -1014,7 +1019,15 @@ cromwell::private::vault_login() {
         cromwell::private::exec_silent_function cromwell::private::vault_login
     elif [[ "${CROMWELL_BUILD_IS_SECURE}" == "true" ]]; then
         case "${CROMWELL_BUILD_PROVIDER}" in
-            "${CROMWELL_BUILD_PROVIDER_TRAVIS}")
+            "${CROMWELL_BUILD_PROVIDER_TRAVIS}"|\
+            "${CROMWELL_BUILD_PROVIDER_CIRCLE}")
+
+                if [[ "${CROMWELL_BUILD_PROVIDER}" == "${CROMWELL_BUILD_PROVIDER_CIRCLE}" ]]; then
+                  VAULT_TOKEN=$( docker run --rm -v "${CROMWELL_BUILD_HOME_DIRECTORY}:/root:rw" \
+                    broadinstitute/dsde-toolbox:dev vault write -field=token auth/approle/login \
```

Comment:
I'm not familiar with how AppRole stuff works, but I'm very surprised to see `vault write` in the test script. I guess that's what this says to do, though: https://www.vaultproject.io/docs/auth/approle. Just pointing it out in case it strikes anyone else as strange.

Reply:
Also here: https://broadworkbench.atlassian.net/browse/DDO-1050?focusedCommentId=43599